src/theory/bv/bitblast/simple_bitblaster.cpp

```diff
@@ -129,6 +129,17 @@ Node BBSimple::getModelFromSatSolver(TNode a, bool fullModel)
   return utils::mkConst(bits.size(), value);
 }
 
+void BBSimple::computeRelevantTerms(std::set<Node>& termSet)
+{
+  for (const auto& var : d_variables)
+  {
+    if (termSet.find(var) == termSet.end())
+    {
+      termSet.insert(var);
+    }
```

Comment:
```suggestion
    termSet.insert(var);
```

Seems a bit redundant to check if `var` is in the set.